lib/fastlane.rb

```diff
@@ -1,4 +1,4 @@
-require 'json'
+require 'fastlane/core_ext/string' # this has to be above most of the other requires
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.